Chamasma/Domain/Sources/Domain/Models/Meal.swift

```diff
@@ -8,38 +8,59 @@
 import SwiftUI
 
 public struct Meal {
+    public let id = UUID()
     public let name: String
     public let description: String
+    public let rating: Double
     public let price: Double
     public let image: Image
     public let ingredients: [String]
     public let isVegetarian: Bool
     public let isVegan: Bool
     public let isGlutenFree: Bool
+    public let isPopular: Bool
     public let isSpicy: Bool
     public let allergens: [Allergen]
 
     public init(
         name: String,
         description: String,
+        rating: Double,
         price: Double,
         image: Image,
         ingredients: [String],
         isVegetarian: Bool,
         isVegan: Bool,
         isGlutenFree: Bool,
+        isPopular: Bool,
         isSpicy: Bool,
         allergens: [Allergen]
     ) {
         self.name = name
         self.description = description
+        self.rating = rating
         self.price = price
         self.image = image
         self.ingredients = ingredients
         self.isVegetarian = isVegetarian
         self.isVegan = isVegan
         self.isGlutenFree = isGlutenFree
+        self.isPopular = isPopular
         self.isSpicy = isSpicy
         self.allergens = allergens
     }
 }
+
+extension Meal: Identifiable, Hashable {
+    var identifier: String {
+        return UUID().uuidString
+    }
+    
+    public func hash(into hasher: inout Hasher) {
+        return hasher.combine(identifier)
```

Comment:
```suggestion
        hasher.combine(identifier)
```

Reply:
instead of implementing custom Hash _function,_ since Meal is struct you can mark sub-properties of it with Hashable